### PR TITLE
Complete select box multi-select

### PR DESCRIPTION
**Pull request by amandeep-singh-vs**
GIF : 
SelectBox

### CODE REVIEW

File: AnimatedComponents/src/components/select/Select.tsx [hunk 1]
```diff
@@ -115,40 +133,92 @@ const Select = (props: ISelectProps) => {
...
-          { top: selectedHeight, height: heightAnim, maxHeight: 200 },
-        ]}>
-        {renderOptions()}
-      </Animated.View>
+      {isOpen && (
+        <Modal
+          transparent
+          animationType="none"
+          visible={isOpen}
+          onRequestClose={() => setIsOpen(false)}>
```
Comment (Pranjul-vs) [inline AnimatedComponents/src/components/select/Select.tsx:206]: remove anonymous arrow functions, check all places.

File: AnimatedComponents/src/components/select/Select.tsx [hunk 2]
```diff
@@ -115,40 +133,92 @@ const Select = (props: ISelectProps) => {
...
+          <TouchableOpacity
+            style={styles.modalOverlay}
+            activeOpacity={1}
+            onPressOut={() => setIsOpen(false)}>
+            <View
+              style={[
+                styles.modalContent,
+                { top: dropdownTop, maxHeight: options.length > 4 ? '55%' : '100%' },
```
Comment (Pranjul-vs) [inline AnimatedComponents/src/components/select/Select.tsx:214]: create a const for this ternary.

File: AnimatedComponents/src/components/select/Select.tsx [hunk 3]
```diff
@@ -115,40 +133,92 @@ const Select = (props: ISelectProps) => {
...
         activeOpacity={10}
-        onLayout={event => setSelectedHeight(event.nativeEvent.layout.height)}
         onPress={handleOpenClosePress}
-        style={[styles.container, style]}>
-        <Animated.Text style={[styles.selectText, { opacity: fadeAnim }]}>
-          {title || placeholderText}
-        </Animated.Text>
+        style={[styles.container, selectContainerStyle]}>
+        <Animated.View
+          style={{
```
Comment (Pranjul-vs) [inline AnimatedComponents/src/components/select/Select.tsx:186]: why are we using inline styles.

File: AnimatedComponents/App.tsx [hunk 4]
```diff
@@ -20,7 +20,7 @@ import type { ISelectOption } from './src/components/select/Select'
 
 const App = () => {
   const [isLoading, setIsLoading] = useState(false)
-  const [select, setSelect] = useState<ISelectOption>({ title: '', value: '' })
+  const [select, setSelect] = useState<ISelectOption | ISelectOption[]>([])
```
Comment (Pranjul-vs) [inline AnimatedComponents/App.tsx:23]: create a demo file and import the component here .

**Review by pranjul-parnami-vs — CHANGES_REQUESTED** [review]

**Pull request merged**